target/linux/mpc85xx/files/arch/powerpc/boot/dts/t30-w.dts

```diff
@@ -0,0 +1,432 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/include/ "fsl/p1020si-pre.dtsi"
+/ {
+	model = "watchguard,t30-w";
+	compatible = "watchguard,t30-w";
+	
+	aliases {
+		serial0 = &serial0;
+		serial1 = &serial1;
+		ethernet0 = &enet0;
+		pci0 = &pci0;
+		pci1 = &pci1;
+		led-boot = &attn_orange;
+		led-failsafe = &status_red;
+		led-running = &attn_orange;
+		led-upgrade = &status_red;
+	};
+		
+	memory {
+		device_type = "memory";
+	};
+
+	lbc: localbus@ffe05000 {
+		reg = <0x0 0xffe05000 0x0 0x1000>;
+		compatible = "fsl,p1020-elbc","fsl,elbc","simple-bus";
+		interrupts = <0x13 0x02>;
+		interrupt-parent = <&mpic>;
+
+		/* NOR Flash */
+		ranges = <0x00 0x00 0x00 0xefc00000 0x100000>;
+		
+		flash@0 {
+			#address-cells = <0x01>;
+			#size-cells = <0x01>;
+			compatible = "cfi-flash";
+			reg = <0x00 0x00 0x400000>;
+			bank-width = <0x02>;
+			device-width = <0x01>;
+			
+			partitions {
+				compatible = "fixed-partitions";
+				#address-cells = <1>;
+				#size-cells = <1>;
+				
+				partition@0 {
+					reg = <0x00 0x20000>;
+					label = "nor-cfg0";
+
+					nvmem-layout {
+						compatible = "fixed-layout";
+						#address-cells = <1>;
+						#size-cells = <1>;
+						macaddr_hwinfo_1830: macaddr@1830 {
+							compatible = "mac-base";
+							reg = <0x1830 0x6>;
```

Comment:
uhhh no. nvmem-cells is missing. the size is also wrong.

Reply:
resolved